pkgs/servers/http/nginx/modules.nix

```diff
@@ -51,8 +51,8 @@
     src = fetchFromGitHub {
       owner = "SpiderLabs";
       repo = "ModSecurity-nginx";
-      rev = "abbf2c47f6f3205484a1a9db618e067dce213b89";
-      sha256 = "04ar51bnqjca6g4p2irymgdmc8rh5nsi8ml43srm4krllnkvw8qn";
+      rev = "a2a5858d249222938c2f5e48087a922c63d7f9d8";
+      sha256 = "1zj0fq35hddzf7b3x40xlbss866lg7w2vd1bbm8g1hcq1ny2s84n";
     };
     inputs = [ pkgs.curl pkgs.geoip pkgs.libmodsecurity pkgs.libxml2 pkgs.lmdb pkgs.yajl ];
```

Comment:
There is no `libmodsecurity` package. Is `modsecurity_standalone` wanted here instead?

Reply:
Need this PR - https://github.com/NixOS/nixpkgs/pull/32075